app/scripts/directives/podMetrics.js

```diff
@@ -358,6 +358,11 @@ angular.module('openshiftConsole')
               // get the unit value if specified
               var unit =  metric.unit || "";
 
+              // A typical metric ID is of the form "pod/<pod-id>/custom/<some-endpoint-info>"
+              // such as "pod/be381d4b-87fc-11e5-b2a3-525400b33d1d/custom/JVM-Heap-Memory-Used".
```

Comment:
The good thing about using `getMetricInfo` is it's the same key MetricsService uses when it creates the map, so they should always match.

https://github.com/openshift/origin-web-console/blob/master/app/scripts/services/metrics.js#L126-L131